lib/phobos/listener.rb

```diff
@@ -9,7 +9,7 @@ class Listener
     DEFAULT_MAX_BYTES_PER_PARTITION = 1_048_576 # 1 MB
     DELIVERY_OPTS = %w[batch message inline_batch].freeze
 
-    attr_reader :group_id, :topic, :id
+    attr_reader :group_id, :topics, :id
```

Comment:
For backwards compatibility, let's keep both `:topic` and `:topics` and deprecate `:topic`. `attr_reader` is public so we have no idea if someone would be accessing this for some reason.